packages/blockchain-wallet-v4-frontend/src/data/networkConfig/slice.ts

```diff
@@ -0,0 +1,31 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+import Remote from '@core/remote'
+
+import { NetworkConfig, NetworkConfigState } from './types'
+
+const initialState: NetworkConfigState = {
+  config: Remote.NotAsked
+}
+
+const networkConfigSlice = createSlice({
+  initialState,
+  name: 'networkConfig',
+  reducers: {
+    fetchNetworkConfig: () => {},
+    fetchNetworkConfigFailure: (state) => {
+      state.config = Remote.Failure(null)
```

Comment:
We do not really care about anything except the fact that there is an error at this stage. I guess could be just `Remote.Failure()`, but I saw usage of both across the codebase. I think we shall fix `Remote` module types first (since they are broken a bit), and then decide on the unified approach.